node/src/main/scala/io/casperlabs/node/configuration/Configuration.scala

```diff
@@ -67,6 +67,7 @@ object Configuration extends ParserImplicits {
       dataDir: Path,
       maxNumOfConnections: Int,
       maxMessageSize: Int,
+      eventStreamBufferSize: Int Refined NonNegative,
```

Comment:
```suggestion
      eventStreamBufferSize: Int Refined Positive,
```
I don't know if a 0 sized buffer is of much use, other than to disable the feature completely.